### DISCUSSION TITLE
Group layout.

### DISCUSSION THREAD

**Discussion by eiLLFire** [fabricjs/fabric.js#10148] [Ideas]
I am using version 6.4.2. I hope for a feature where I can input a gap, and the child elements will be responsively distributed, similar to the CSS flex layout.
```ts
    const group = new Group(activeObjects, {
      gap: 12
    })
```
![GAP](https://github.com/user-attachments/assets/57e96ee6-0ddb-4a3a-b92b-b430aa2f51e2)
Can the LayoutManager achieve this feature? I rarely find articles explaining LayoutManager.

**Comment by asturur** [comment]
this because the GroupLayout is a beta feature that was done to accomodate some desire to change.
The layout manager is supposed to be a skeleton on which you implement your preferred version of it.
You can develop a flex layout manager and either keep it for you or contribute it back for the extensions bundle
